convert2rhel/checks.py

```diff
@@ -495,15 +506,33 @@ def is_loaded_kernel_latest():
     """Check if the loaded kernel is behind or of the same version as in yum repos."""
     logger.task("Prepare: Checking if the loaded kernel version is the most recent")
 
+    reposdir = get_hardcoded_repofiles_dir()
```

Comment:
```suggestion
    LATEST_RHEL_8_MINOR_VERSION = 6
    if system_info.id == "oracle" and system_info.version.major == 8 and system_info.version.minor < LATEST_RHEL_8_MINOR_VERSION:
        logger.info("Skipping the check because there are no publicly available %s %d.%d repositories available."
        % (system_info.name, system_info.version.major, system_info.version.minor))
        return
    reposdir = get_hardcoded_repofiles_dir()
```
@r0x0d, @abadger, we may improve hardcoding the latest minor version by employing scheduled RHEL release dates. But we might do that in a separate PR.

Or shall we perhaps simply skip this check on OL 8, no matter the minor version, to save ourselves from the hassle with hardcoding the minor versions?